src/web/js-interop.md

```diff
@@ -22,3 +22,77 @@ For help using the `js` package, see the following:
 [js-api]: {{site.pub-api}}/js
 [sass]: {{site.pub-pkg}}/sass
 
+## Future-proof JS interop
+
+Dart's JS interop story is currently evolving. 
+Many of the features that enable future JS interop,
+and support of all three compilers,
```

Comment:
For someone first reading this "all three compilers" may be a bit confusing, as they may think of non-web compilers or not know the difference or existence of the different web compilers.

Maybe something "all web compilers, including Dart's in-progress Wasm compiler," as that seems to be the important part of this section.

Reply:
I was hoping someone would catch this, because _I_ don't know what "all three compilers" means, I think I got it from one of the related issues or issue comments. What are all three? 

I would rather say something like 
> and support of web compilers, including ________, _________, _and_ Dart's in-progress Wasm compiler"